__tests__/client.test.js

```diff
@@ -186,4 +186,12 @@ describe("BncClient test", async () => {
     const res1 = client.createAccountWithKeystore("12345678")
     expect(res1.address.includes('bnb')).toBeTruthy()
   })
+
+  it("get markets works", async () => {
+    const client = await getClient(false)
+    const { result: markets, status } = await client.getMarkets(150)
+
+    expect(markets.length).toBe(150)
```

Comment:
This test could fail if the number of available markets is less than 150. 

I would recommend checking that `length > 0` and that the first market contains expected fields (`base_asset_symbol`, `quote_asset_symbol`, `price`, etc).